tests/installation/bootloader_s390.pm

```diff
@@ -194,7 +194,7 @@ EO_frickin_boot_parms
 
     my $output_delim
       = $display_type eq "SSH" || $display_type eq "SSH-X" ? qr/\Q***  run 'yast.ssh' to start the installation  ***\E/
-      : $display_type eq "VNC" ? qr/\Q*** Starting YaST2 ***\E/
+      : $display_type eq "VNC" ? qr/\Q*** Starting YaST(2|) ***\E/
```

Comment:
does it work between \Q AND \E ?
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15701#discussion_r995347582

Reply:
https://openqa.nue.suse.com/t9725192
https://openqa.nue.suse.com/t9725191
https://openqa.nue.suse.com/t9725190
https://openqa.nue.suse.com/t9725189

https://openqa.suse.de/t9725188

Reply:
> does it work between \Q AND \E ? [#15701 (comment)](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15701#discussion_r995347582)
https://openqa.nue.suse.com/t9725192  and https://openqa.nue.suse.com/t9725191  are failed.